.devcontainer/devcontainer.json

```diff
@@ -20,7 +20,7 @@
 	"remoteUser": "node",
 	
 	"features": {
-		"docker-in-docker": "latest"
+		"ghcr.io/devcontainers/features/docker-in-docker:1": {}
```

Comment:
`"latest"` would still work, wouldn't it?

Reply:
`latest` is implied as the [default value from the `devcontainer-feature.json`](https://github.com/devcontainers/features/blob/main/src/docker-in-docker/devcontainer-feature.json#L15).  It is redundant to add it here.  (And additionally, to a new user it is not clear that this value maps to the version of _docker_).  I prefer the empty object as it indicates "use the defaults the author has set for me"